rubygem-foreman_maintain/rubygem-foreman_maintain.spec

```diff
@@ -2,13 +2,13 @@
 %{!?scl:%global pkg_name %{name}}
 
 %global gem_name foreman_maintain
-%global confdir foreman_maintain
+%global package_name foreman-maintain
```

Comment:
Oh, I do see your point @iNecas . And I think you logic makes sense. @ekohl any thoughts?